src/ui/FacetValueSuggestions/FacetValueSuggestions.ts

```diff
@@ -143,8 +148,8 @@ export class FacetValueSuggestions extends Component {
   private queryStateFieldFacetId;
 
   static defaultTemplate(this: FacetValueSuggestions, row: IFacetValueSuggestionRow, omnibox: Omnibox): string {
-    const keyword = DomUtils.highlightElement(row.keyword, omnibox.getText(), 'coveo-omnibox-hightlight2');
-    const facetValue = DomUtils.highlightElement(row.value, row.value, 'coveo-omnibox-hightlight');
+    const keyword = row.keyword.html;
+    const facetValue = DomUtils.highlight(row.value, 'coveo-omnibox-hightlight');
```

Comment:
🎊 amazing

Even though it touches more methods, in the end, it makes it so much simpler here! Good job!